sunpy/net/vso/vso.py

```diff
@@ -271,9 +271,12 @@ def response_block_properties(self):
         """
         s = {a if not a.startswith('_') else None for a in dir(self[0])}
         for resp in self[1:]:
+            if len(s) == 0:
```

Comment:
Could this check actually be moved up outside of the for loop?

```python
s = ...
if not s:
    return
```

Reply:
no, because at first, the set will contain some elements it's after the first intersection that set may become empty.

Reply:
no, because set may not be empty at the start but may become empty after the first or second iteration.